{"name": "test_call_api_with_error_status_codes[400-API调用失败：400 Client Error]", "status": "passed", "description": "测试各种HTTP错误状态码", "parameters": [{"name": "status_code", "value": "400"}, {"name": "expected_exception_msg", "value": "'API调用失败：400 Client Error'"}], "start": 1761614695510, "stop": 1761614695513, "uuid": "26d4b186-9a36-44c9-baba-96142e47baf5", "historyId": "59811a3be45070070a0e0b8905b59817", "testCaseId": "0ba28f683659835500083ccde95cdb5e", "fullName": "testcases.test_error_response.TestCallApi#test_call_api_with_error_status_codes", "labels": [{"name": "parentSuite", "value": "testcases"}, {"name": "suite", "value": "test_error_response"}, {"name": "subSuite", "value": "TestCallApi"}, {"name": "host", "value": "DESKTOP-SQEMBVG"}, {"name": "thread", "value": "15580-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcases.test_error_response"}]}